### PR TITLE
Added ad/freebird60

### PR DESCRIPTION
**Pull request by KnoblesseOblige**
## Description

Added Freebird60 keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebsforall/freebird60/readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Freebird60
+
+![freebird60](imgur.com image replace me!)
```
Comment (drashna) [inline keyboards/keebsforall/freebird60/readme.md:3]: Got an image?

Comment (KnoblesseOblige) [reply keyboards/keebsforall/freebird60/readme.md]: That's been added.

File: keyboards/keebsforall/freebird60/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Freebird60
+
+![freebird60](https://imgur.com/a/cfYz93b)
+
+*A 60% hotswap keyboard with mutliple layout compatibility*
```
Comment (fauxpark) [inline keyboards/keebsforall/freebird60/readme.md:5]:
```suggestion
A 60% hotswap keyboard with multiple layout compatibility.
```

File: keyboards/keebsforall/freebird60/readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Freebird60
+
+![freebird60](https://imgur.com/a/cfYz93b)
```
Comment (fauxpark) [inline keyboards/keebsforall/freebird60/readme.md:3]:
```suggestion
![freebird60](https://i.imgur.com/qGtVVbyl.jpeg)
```
Needs to be a direct link.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/keebsforall/freebird60/readme.md (hunk 3 above)
Comment (KnoblesseOblige) [reply keyboards/keebsforall/freebird60/readme.md:3]: Apologies. I had pasted that in to the file and didn't save before committing. LGTM

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**